packages/manager/modules/emailpro/src/account/delegation/emailpro-account-delegation.html

```diff
@@ -227,23 +227,6 @@
                             />
                         </td>
                     </tr>
-
-                    <tr
-                        data-ng-repeat="account in accounts.list.results track by account.id | orderBy:'id':false"
```

Comment:
Cannot work because it's already nested into a ngRepeat.

See: https://github.com/ovh/manager/pull/3460/files#diff-e7d7f6886129a25551f1a3fac9a9147bL115